tshirts/tshirts.java

```diff
@@ -9,11 +9,16 @@ static String size(int cms) {
         }
     }
 
-    public static void main(String[] args) { 
-        assert(size(37) == "S");
-        assert(size(38) == "M");
-        assert(size(40) == "M");
-        assert(size(43) == "L");
-        System.out.println("All is well (maybe!)");
+    public static void main(String[] args) {
+            assert (size(37) == "S");

```

Comment:
Good coverage of test values 👍 
42 is a boundary value. Test must be present for all boundary values.